### ISSUE TITLE
Bug Report: Cleared % goes down if you get a new highscore but not a full combo

### ISSUE THREAD

**Issue by JackXson-Real** [FunkinCrew/Funkin#2582]
[weed]: <> (FILL THIS ISSUE THING OUT AS MUCH AS POSSIBLE)
[weed]: <> (OR ELSE YOUR ISSUE WILL BE LESS LIKELY TO BE SOLVED!)
[weed]: <> (DO NOT POST ABOUT ISSUES FROM OTHER FNF MOD ENGINES! I CANNOT AND PROBABLY WON'T SOLVE THOSE!)
[weed]: <> (GO TO THEIR RESPECTIVE GITHUB ISSUES AND REPORT THEM THERE LOL!)

[weed]: <> (ALSO MAKE SURE THAT YOU USE PROPER LABELS, IF YOU'RE RUNNING INTO COMPILER ISSUES, USE THE compiler issue LABEL!!!)

### If you are playing the game in a browser, what site are you playing it from?

[weed]: <> (Put an X in the [ ] thingies to fill out checkbox!)
[weed]: <> (something like [x] pretty much, don't screw up or you will look stupid)

- [ ] [Newgrounds](https://www.newgrounds.com/portal/view/770371)
- [x] [Itch.io](https://ninja-muffin24.itch.io/funkin)? Specify below
- - [x] Windows
- - [ ] Mac
- - [ ] Linux

### If you are playing the game in a browser, what browser are you using?

[weed]: <> (Again, put an x in the [ ] box!)

- [ ] Google Chrome (or chomium based like Brave, vivaldi, MS Edge)
- [ ] Firefox
- [ ] Safari

## What version of the game are you using? Look in the bottom left corner of the main menu. (ex: 0.2.7, 0.2.1, shit like that)
0.3.3

## Have you identified any steps to reproduce the bug? If so, please describe them below in as much detail as possible. Use images if possible.
Have a cleared 100% on a level, then get a new highscore, but not an FC.

## Please describe your issue. Provide extensive detail and images if possible.
If you have a 100% cleared on a song, but then get a new highscore, but you also got some bads, shits, or misses, the cleared % of a song will go down to 99. I'm not sure if this is a bug or not, but I find it kind of weird that the highscore and cleared % are connected. On top of that, it seems even if I do get an FC, my cleared % won't go back up to 100% unless I get another high score. I kind of wish that I could keep the 100% cleared without having to worry about it resetting if I get a new highscore.

**Comment by EliteMasterEric** [comment]
You're talking about the clear % displayed in the Freeplay menu? That's bugged I think, it sometimes it stops 1 below the actual value it should display when it scrolls around.

I think it's fixed in this next update.

**Comment by JackXson-Real** [comment]
> You're talking about the clear % displayed in the Freeplay menu? That's bugged I think, it sometimes it stops 1 below the actual value it should display when it scrolls around.
> 
> I think it's fixed in this next update.

Yea I’m talking about that. I swear the problem is the one I mentioned though. It consistently showed the correct % (100) until I got a new high score without having a full combo, and now it consistently shows 99%. I also thought the issue where it randomly goes down 1% has been fixed already since I haven’t experienced that since updating to 0.3.3.
